superset-frontend/spec/javascripts/datasource/fixtures.tsx

```diff
@@ -29,7 +29,8 @@ export const columns: ColumnMeta[] = [
     id: 516,
     is_dttm: false,
     python_date_format: null,
-    type: ColumnType.DOUBLE,
+    type: 'DOUBLE',
```

Comment:
Why don't we use `ColumnType` here anymore? Comment applies to all other "ColumnType -> hardcoded string" changes

Reply:
`ColumnType` has been removed from `superset-ui/core` and replaced with `string`, as it's the raw column type coming back from the database. `ColumnType` was really just a generalization that mostly complied with ANSI SQL types, but this was not a scalable solution, as the frontend mostly isn't aware of which database is being used. As an example, the datatype `DOUBLE PRECISION` wasn't being picked up on the frontend due to it being a datatype that's mostly only available on dialects of Postgres. In corner cases two datatypes with the same name can mean different things on different databases (the boolean type on MySQL comes to mind, which is actually an integer). As these are now handled in the backend and mapped to `GenericDataType`, that's what we're now going to be using for the most part.